.teamcity/_self/projects/WebApp.kt

```diff
@@ -903,3 +904,154 @@ object RunCalypsoPlaywrightE2eDesktopTests : BuildType({
 		}
 	}
 })
+
+object RunCalypsoPlaywrightE2eMobileTests : BuildType({
```

Comment:
This is fine to leave at it is for now, but eventually we should look at extracting this to a function to avoid so much repetition with `RunCalypsoPlaywrightE2eDesktopTests` (same with `RunCalypsoE2eDesktopTests`+`RunCalypsoE2eMobileTests`).

Marcelo did an awesome job in https://github.com/Automattic/wp-calypso/blob/trunk/.teamcity/_self/projects/WPComTests.kt#L41

Reply:
Gotcha. I will make another task for that.

Reply:
Filed #54812 for this.